  * debian/patches/greek.diff: New patch to replace dead_horn and
    dead_ogonek by U0313/U0314, which are the correct breathing
    signs for polytonic Greek.  This change can be performed now
    that an updated el_GR.UTF-8/Compose file hit testing.
    Closes: #386385  Thanks Jan Willem Stumpel.

Not submitted upstream.  This patch can be submitted only when
el_GR.UTF-8/Compose contains rules with U0313/U0314.

Index: xkb-data/symbols/gr
===================================================================
--- xkb-data.orig/symbols/gr
+++ xkb-data/symbols/gr
@@ -113,8 +113,8 @@
     key <AD11> { [ 	dead_tilde,	dead_diaeresis,	dead_macron	] };
     key <AD12> { [	dead_iota,	VoidSymbol,	dead_breve	] };
 
-    key <AC10> { [	dead_acute,		dead_horn	] };
-    key <AC11> { [	dead_grave,		dead_ogonek	] };
+    key <AC10> { [	dead_acute,		U0313	] };
+    key <AC11> { [	dead_grave,		U0314	] };
 
 };
 
